src/calendar.rs

```diff
@@ -62,11 +62,13 @@ pub(crate) fn time_from_ymdhms_utc(
     ))
 }
 
+const UNIX_EPOCH_YEAR: u64 = 1970;
```

Comment:
Style nit: `const` below the function rather than above it?

Reply:
Fixed in [06674b1](https://github.com/rustls/webpki/pull/71/commits/06674b1b70179dffe53b4a88aef88f7b088f311c).